swift/Samples/TicTacToe/Sources/Game/GamePlayScreen.swift

```diff
@@ -154,9 +153,9 @@ final class GamePlayViewController: ScreenViewController<GamePlayScreen> {
     }
 
     @objc private func buttonPressed(sender: UIButton) {
-        for row in 0..<3 {
+        for row in 0 ..< 3 {
             let cols = cells[row]
-            for col in 0..<3 {
```

Comment:
I agree with the space between the operator and the number but it was nice to see a heart in the code once in a while 😄

Reply:
I think this is a bit awkward for ranges actually, `0..<3` is cleaner than `0 ..< 3` IMO

Reply:
I think the Google Style guide recommends no space in this case. Unless anybody objects, I'll disable the rule for this case.

Reply:
This is the `spaceAroundOperators` rule. Even if this is not the recommendation for ranges - I'd like the keep `spaceAroundOperators` rule. Any strong objections to keeping this rule enabled?